ReactiveObjCTests/RACNeverSignalSpec.m

```diff
@@ -41,6 +41,22 @@
   }
 
   expect(@(disposable.isDisposed)).to(beFalsy());
+
+  [disposable dispose];
+});
+
+qck_it(@"should not dispose disposable if returned disposable is weakly held", ^{
+  __weak RACDisposable *disposable;
+
+  @autoreleasepool {
+    RACSignal *signal = [RACNeverSignal never];
+    disposable = [signal subscribeNext:^(id _Nullable x) {}];
+  }
+
+  expect(disposable).notTo(beNil());
+  expect(@(disposable.isDisposed)).to(beFalsy());
+
+  [disposable dispose];
 });
 
 qck_it(@"should deallocate subscriber on subscription disposal", ^{
```

Comment:
we can also check very similarly the weak subscriber is not deallocated if `dispose` was not called yet